rcgen/src/string_types.rs

```diff
@@ -0,0 +1,490 @@
+use crate::Error;
+use std::str::FromStr;
+
+/// ASN.1 `PrintableString` type.
+///
+/// Supports a subset of the ASCII character set (described below).
+///
+/// For the full ASCII character set, use
+/// [`Ia5String`][`crate::Ia5String`].
+///
+/// # Examples
+///
+/// You can create a `PrintableString` from [a literal string][`&str`] with [`PrintableString::try_from`]:
+///
+/// ```
+/// use rcgen::PrintableString;
+/// let hello = PrintableString::try_from("hello").unwrap();
+/// ```
+///
+/// # Supported characters
```

Comment:
Maybe it would make sense to link to the [PrintableString](https://en.wikipedia.org/wiki/PrintableString) wikipedia page here and also say that this is a true subset of the ascii printable characters? Say @ is a printable charater as per ASCII but can't be part of ASN.1's `PrintableString`.

Reply:
That's much clearer, thank you! I've updated the doc